uberfire-m2repo-editor/uberfire-m2repo-editor-backend/src/main/java/org/guvnor/m2repo/backend/server/FixNotMavenizedArtifactInfo.java

```diff
@@ -0,0 +1,61 @@
+package org.guvnor.m2repo.backend.server;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import org.guvnor.common.services.project.model.GAV;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.ARTIFACT_ID;
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.GROUP_ID;
+import static org.guvnor.m2repo.backend.server.M2ServletContextListener.VERSION;
+
+public class FixNotMavenizedArtifactInfo {
+
+    private static final Logger logger = LoggerFactory.getLogger(FixNotMavenizedArtifactInfo.class);
+
+    private final Properties notMavenizedArtifacts = new Properties();
+    private final String pomTemplate;
+
+    public FixNotMavenizedArtifactInfo() {
+        String pomTemplate = null;
+        try {
+            pomTemplate = new BufferedReader(new InputStreamReader(M2ServletContextListener.class.getResourceAsStream("template.pom")))
```

Comment:
Any reason to split an then immediately join using newline? Why not just load the artifact as a string?

Also in this case (since we only have one fixed template which is relatively small) I'd simplify the code and include the template as a `private static final String` field into this class directly.